AsyncDisplayKit/ASImageNode+AnimatedImage.mm

```diff
@@ -85,6 +85,23 @@ - (void)coverImageCompleted:(UIImage *)coverImage
   }
 }
 
+- (NSString *)animatedImageRunLoopMode
+{
+  ASDN::MutexLocker l(_displayLinkLock);
+  return _animatedImageRunLoopMode;
+}
+
+- (void)setAnimatedImageRunLoopMode:(NSString *)runLoopMode
+{
+  ASDN::MutexLocker l(_displayLinkLock);
+
+  if (_displayLink != nil) {
+    [_displayLink removeFromRunLoop:[NSRunLoop mainRunLoop] forMode:_animatedImageRunLoopMode];
+    [_displayLink addToRunLoop:[NSRunLoop mainRunLoop] forMode:runLoopMode];
```

Comment:
@appleguy A nil check was added, and the default runloop mode was moved to a constant. 

As far as changing the default: I personally assume that when people see GIFs stop during scrolling/zooming they would assume it's a bug (videos don't display the same behavior). At least, this is what I observed when I implemented a similar runloop configuration in a different animated gif library back in 2013: [https://github.com/dtorres/OLImageView/pull/8] caused issues: [https://github.com/dtorres/OLImageView/issues/27] [https://github.com/dtorres/OLImageView/issues/42]. That being said, I'm fine with whatever default mode is chosen.